samples/app/MeshAssimp.cpp

```diff
@@ -289,9 +577,23 @@ bool MeshAssimp::setFromFile(const Path& file,
                     size_t indicesOffset = outPositions.size();
 
                     for (size_t j = 0; j < numVertices; j++) {
-                        quatf q = mat3f::packTangentFrame({tangents[j], bitangents[j], normals[j]});
+                        float3 normal = normals[j];
+                        float3 texCoord = texCoords ? texCoords[j] : float3{0.0};
+                        float3 tangent;
+                        float3 bitangent;
+
+                        //If the tangent and bitangent don't exist, make arbitrary ones
```

Comment:
We should at least mention mikktspace here since that's what the spec recommends, and that's what everybody uses.

Reply:
What should we say about mikktspace?

Reply:
Maybe something like:

TODO: The glTF specification recommends using the MikkTSpace algorithm for computing tangent vectors in the absence of explicit tangents.